TEMPLATE.md

```diff
@@ -0,0 +1,652 @@
+## Shortcuts from raulfraile, applied on Phpstorm by Pouzor ##
+
+All shortcuts start with the `sf` prefix and are both short and intuitive:
+
+### Controller ###
+
+`sfcontroller`
+
+``` php
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    public function indexAction()
+    {
+        return $this->render('.html.twig');
+    }
+}
+```
+
+`sfcontrollera`
+
+``` php
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class DefaultController extends Controller
+{
+    /**
+     * @Route("/", name="")
+     * @Template()
+     */
+    public function indexAction()
+    {
+
+    }
+}
+```
+
+`sfaction`
+
+``` php
+public function indexAction()
+{
+    return $this->render('.html.twig');
+}
+```
+
+`sfactiona`
```

Comment:
why "a" for the extra character? I think the be most portable, we should *just* have `sfaction`, and have them use another live template if they want to add the `@Route` above it